File: platforms/android/assets/www/js/controllers/SilviaVincenzo/lista_prodottiController.java

myApp.controller('lista_prodottiController', function($scope,$http, localDbService) { 

	console.log("lista prodotti ricercati");
	
	 $scope.lista_prodotti= function(prodottoTrovato){
		 console.log('lista prodotti');
		 var prodottoTrovato
		 var prodottoTrovato = {
				    "immagine": prodottoTrovato.immagine,
				    "nome": prodottoTrovato.nome,
				    "descrizione": prodottoTrovato.descrizione,
				//manca nome metodo recpuero prodotto 
				//remotoApiService.listaProdotti(prodottoTrovato);
				
				}
		}

});   
	

File: platforms/android/assets/www/js/controllers/SilviaVincenzo/ListaProdottiController.js
myApp.controller('listaProdottiController', function($scope,$http,prodottoService) {

	console.log("listaProdottiController");
	$http.get("http://joyfoodamministratore-sisp.rhcloud.com/public/listAllProdotti")

	.then(function(response) {
		console.log('prodotti = '+response.data.length);
		
		$scope.prodotti = response.data;
		
	});

	$scope.changeView=function(prodotto){
	    prodottoService.saveProdotto(prodotto);
	    console.log(prodotto.id);
	    myNavigator.pushPage("html/SilviaVincenzo/dettaglio_prodotto.html");
	}

});